security/citadel/templates/deployment.yaml

```diff
@@ -77,11 +82,22 @@ spec:
           {{- end }}
           {{- end }}
           livenessProbe:
+          {{- if .Values.citadelHealthCheck }}
```

Comment:
Can you have two liveness probes like this? It seems like this will break

Reply:
I think we only have one single liveness probe at a time.
If the citadelHealthCheck is true, use the one we built for Citadel. Otherwise (line 94), use the default one (which doesn't check the sanity of the internal function).

Reply:
Oh I missed the `else` part of this